wls-gui-wahllokalsystem/tests/composables/ergebnisermittlung/obwStapelCUtils.spec.ts

```diff
@@ -590,30 +590,38 @@ describe("obwStapelCUtils", () => {
     });
   });
 
-  describe("getErgebnisStapelCByWahlvorschlagIdOrZero", () => {
-    it("should_returnErgebnisStapelC_when_stapelCGueltigErgebnisseHasEntryForWahlvorschlagID", () => {
+  describe("getStapelCGueltigErgebnisseByWahlvorschlagIdOrZero", () => {
+    it("should_returnStapelCGueltigErgebnis_when_stapelCGueltigErgebnisseHasEntryForWahlvorschlagID", () => {
       const ergebnis1 = prepareErgebnis()
         .ergebnis(generateRandomNumber(4))
         .wahlvorschlagID(generateRandomString(8))
         .build();
       const ergebnis2 = prepareErgebnis()
+        .ergebnis(generateRandomNumber(4))
+        .wahlvorschlagID(ergebnis1.wahlvorschlagID)
+        .build();
+      const ergebnis3 = prepareErgebnis()
         .ergebnis(generateRandomNumber(4))
         .wahlvorschlagID(generateRandomString(8))
         .build();
 
       mockDefinitions.getErgebnisseAndCreateIfMissing.mockReturnValue(
-        prepareErgebnisse().ergebnisse([ergebnis1, ergebnis2]).build()
-      );
-
-      const result1 = unitUnderTest.getErgebnisStapelCByWahlvorschlagIdOrZero(
-        ergebnis1.wahlvorschlagID
-      );
-      const result2 = unitUnderTest.getErgebnisStapelCByWahlvorschlagIdOrZero(
-        ergebnis2.wahlvorschlagID
+        prepareErgebnisse()
+          .ergebnisse([ergebnis1, ergebnis2, ergebnis3])
+          .build()
       );
 
-      expect(result1).toStrictEqual(ergebnis1.ergebnis);
-      expect(result2).toStrictEqual(ergebnis2.ergebnis);
+      const result1 =
+        unitUnderTest.getStapelCGueltigErgebnisseByWahlvorschlagIdOrZero(
+          ergebnis1.wahlvorschlagID
+        );
+      const result2 =
+        unitUnderTest.getStapelCGueltigErgebnisseByWahlvorschlagIdOrZero(
+          ergebnis3.wahlvorschlagID
+        );
+
+      expect(result1.length).toStrictEqual(2);
+      expect(result2.length).toStrictEqual(1);
```

Comment:
```suggestion
      expect(result1.length).toStrictEqual(2);
      expect(result2.length).toStrictEqual(1);
      for (const ergebnis of result1) {
        expect(ergebnis.ergebnis.wahlvorschlagID).toStrictEqual(
          ergebnis1.wahlvorschlagID
        );
      }
      for (const ergebnis of result2) {
        expect(ergebnis.ergebnis.wahlvorschlagID).toStrictEqual(
          ergebnis3.wahlvorschlagID
        );
      }
```

das würde ich der vollständigkeit halber noch ergänzen, da die funktion ja ergebnisse zurückliefert und die length nur in unserem usecase betrachtet wird